modules/kmm-build-validation-stage.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * updating/kmm-preflight-validation.adoc
+
+:_content-type: CONCEPT
+[id="kmm-build-validation-stage_{context}"]
+= Build validation stage
+
+Build validation is executed only when image validation has failed and there is a `build` section in the `Module` that is relevant for the upgraded kernel. Build validation attempts to run the build job and validate that it finishes successfully.
+
+[NOTE]
+====
+You need to specify the kernel version when running `depmod`, as shown here:
+[source,terminal]
+----
+$ RUN depmod -b /opt ${KERNEL_VERSION}.
```

Comment:
Just checking whether this `.` on the end is supposed to be there.

Reply:
It's not. I'll remove it. Thx.